common/importer/chrome_importer_utils_unittest.cc

```diff
@@ -6,56 +6,84 @@
 #include "brave/common/importer/chrome_importer_utils.h"
 #include "base/files/file_path.h"
 #include "base/files/file_util.h"
+#include "base/files/scoped_temp_dir.h"
 #include "base/json/json_reader.h"
 #include "base/path_service.h"
 #include "brave/common/importer/importer_constants.h"
 #include "brave/components/constants/brave_paths.h"
+#include "chrome/common/importer/importer_data_types.h"
 #include "testing/gtest/include/gtest/gtest.h"
 
-namespace {
-const char kTestExtensionsPreferencesFile[] =
-    "Secure_Preferences_for_extension_import";
-base::FilePath GetTestProfilePath() {
-  base::FilePath test_dir;
-  base::PathService::Get(brave::DIR_TEST_DATA, &test_dir);
-  return test_dir.AppendASCII("import").AppendASCII("chrome").AppendASCII(
-      "default");
-}
-// This sample prefs file is fetched after installing two extensions and one
-// theme from webstore with fresh profile.
-base::FilePath GetTestPreferencesPath() {
-  base::FilePath test_dir;
-  base::PathService::Get(brave::DIR_TEST_DATA, &test_dir);
-  return test_dir.AppendASCII("import")
-      .AppendASCII("chrome")
-      .AppendASCII("default")
-      .AppendASCII(kTestExtensionsPreferencesFile);
+class BraveChromeImporterUtilsTest : public testing::Test {
+ public:
+  BraveChromeImporterUtilsTest() {}
+
+  void SetUp() override {
+    EXPECT_TRUE(brave_profile_dir_.CreateUniqueTempDir());
+    base::CreateDirectory(GetTestProfilePath());
+  }
+  base::FilePath GetTestProfilePath() {
+    return brave_profile_dir_.GetPath().AppendASCII("Chrome").AppendASCII(
+        "Default");
+  }
+  void CopyTestFileToProfile(const std::string& source,
+                             const std::string& target) {
+    base::FilePath test_data_dir;
+    base::PathService::Get(brave::DIR_TEST_DATA, &test_data_dir);
+    base::CopyFile(test_data_dir.AppendASCII("import")
+                       .AppendASCII("chrome")
+                       .AppendASCII("default")
+                       .AppendASCII(source),
```

Comment:
Should append `source`? below test preferences are located under `test/data/import/chrome/default/`

Reply:
it doesnt matter actually, we just need a separate folder for files

Reply:
Oops! it was `source` not `"source"` NVM